### PR TITLE
Fix SA tests due to recent changes

### PR DESCRIPTION
**Pull request by lcyyork**
## Description
Fix the SA tests that are not tested by travis.

## User Notes
- [ ] Features added
- [ ] Changes to compilation (if any)

## Checklist
- [ ] Added tests of new features
- [ ] Removed comments in code and input files
- [ ] Documented source code
- [ ] Checked for redundant headers
- [ ] Checked for consistency in the formatting of the output file
- [ ] Documented new features in the manual
- [x] Ready to go!

### CODE REVIEW

**Comment by lcyyork** [comment]
I am not sure why fci-6 is super slow (~10 min) now, but it used to be relatively fast (~3 min).

File: tests/methods/df-dsrg-mrpt2-7-localized-actv/input.dat [hunk 1]
```diff
@@ -58,36 +58,22 @@ set forte{
...
-#  rotate_mos          []
-# # fcimo_localize_actv true
-#}
-#Ecas, wfn = energy('forte', ref_wfn=wfn, return_wfn=True)
-#compare_values(r_avg_cas, Ecas, 8, "SA-CASSCF energy from FORTE FCIMO")
-#cubeprop(wfn)
-
 set forte{
-  job_type newdriver
+  job_type            newdriver
```
Comment (jeffschriber) [inline tests/methods/df-dsrg-mrpt2-7-localized-actv/input.dat]: Is ``newdriver`` needed here?

Comment (lcyyork) [reply tests/methods/df-dsrg-mrpt2-7-localized-actv/input.dat]: Yes, because the job_type is set by the previous block.

**Comment by lcyyork** [comment]
I am not sure why aci-14 fails for travisci. This test case passes on my computer with max_det_orb=128.

**Comment by fevangelista** [comment]
I have disabled test case `ACI-14` since it is not relevant to this PR and created a new branch/PR that will deal with it (see #128).

**Pull request merged**